recipes/_configure.rb

```diff
@@ -7,7 +7,11 @@
 
 # Verify that clusters are configured
 if node['splunk']['node_type'] != :license_server && node['splunk']['config']['clusters'].empty?
-  throw 'You need to configure at least one cluster databag.'
+  if node['splunk']['node_type'] == :forwarder
+    Chef::Log.warn 'You need to configure at least one cluster databag.'
```

Comment:
The wording here could be better.  Maybe something along the lines of "No cluster data bag configured, ensure your outputs are configured elsewhere."